src/ansys/fluent/core/solver/flobject.py

```diff
@@ -925,7 +922,6 @@ def get_root(flproxy) -> Group:
         cls = settings.root
     except Exception:
```

Comment:
Looking at the complete code for this function it seems not right:
```
    try:
        from ansys.fluent.core.solver import settings

        if settings.SHASH != _gethash(obj_info):
            LOG.warning(
                "Mismatch between generated file and server object "
                "info. Dynamically created settings classes will "
                "be used."
            )
            raise RuntimeError("Mismatch in hash values")
        cls = settings.root
    except Exception:
        cls = get_cls("", obj_info)
```
It raises a `RuntimeError` which will force it to go into the Exception handler case right?  If so, should line 923 not be
```suggestion
    except RuntimeError:
```
as it is now, all exceptions are caught and it's not clear from the code in the try block if all possible exceptions could be thrown.

Reply:
Hi Dan, Yeah, sure. Thank you.

Reply:
If you want to merge this code as-is and do it later it's fine.  Just to be clear.

I only noticed this because I ran `pylint` on the module locally and it highlights the issue right.

Reply:
Hi Dan, 
I am thinking to implement the suggestion simultaneously for both Fluent and pyfluent. 
Thank you.

Reply:
Hi Dan,
Sean and I have gone through the code, we haven't found other specific spots.  We have replaced Exception with RuntimeError as per above suggestion. Can you please mention the other specific spots?
Thank you.

Reply:
How about at lines 699, 714, 724, 855 and 924 (924 is the one that I questioned the most since the try block specifically throws a RuntimeError).  In all those cases the implementation will capture every possible exception but what is best is if you specifically catch the exceptions that can actually occur at those locations.  It's ok with me if you want to leave this for now and come back to it later, just something that got pointed out by `pylint`.

Reply:
Hi Dan, Thank you.